format/format.go

```diff
@@ -41,7 +41,8 @@ type Options struct {
 	// not use language features which could break programs.
 	LangVersion string
 
-	ExtraRules bool
+	ExtraRules     bool
+	GofumptVersion string
```

Comment:
why is gofumpt's version a parameter? this seems wrong from an API point of view, as noone should be telling the library what version it is.

if what you need to do is give the format library access to the version string, I'd suggest to move that logic from the main package into the format package. Or even better: move it to an internal package with an exported function or variable, so that both packages can access it without us making that available to third parties directly.